index.html

```diff
@@ -203,7 +203,12 @@ <h2>
           <ol>
             <li>Set |image|'s [=image resource/sizes=] to the result of parsing
```

Comment:
Maybe we should say: 

```suggestion
            <li>Set |image|'s [=image resource/sizes=] to the result of [=link type "icon"|parsing=]
```

Instead... then the parsing algorithm is explicitly linked here and below and we don't need the note.

Reply:
That worked, thanks!